pandas/io/excel/_base.py

```diff
@@ -366,6 +366,10 @@ def _workbook_class(self):
     def load_workbook(self, filepath_or_buffer):
         pass
 
+    @abc.abstractmethod
```

Comment:
Mmm does this have to be abstract? That may be too large a change for a point release.

Reply:
And it seems like several children just pass, so can we just make this non-abstract?

Reply:
OK, will fix